charts/ingress-nginx/tests/controller-daemonset_test.yaml

```diff
@@ -208,3 +208,49 @@ tests:
       - equal:
           path: spec.template.spec.runtimeClassName
           value: myClass
+
+  - it: should create a DaemonSet with resizePolicy if `controller.resizePolicy` is set and Kubernetes version >= 1.30
+    capabilities:
+      majorVersion: 1
+      minorVersion: 30
```

Comment:
Have you checked if this is required? I am actually not 100% sure, I can remember we needed that in some other test, but I can barely remember without checking.